src/components/carousel/index.tsx

```diff
@@ -87,11 +87,11 @@ const Carousel: FC<Props> = (props) => {
   );
   const onClick = useCallback(
     (index: number) => {
-      if (onSlideClick && mainCarousel && mainCarousel.clickAllowed()) {
```

Comment:
`clickAllowed()` is now handled automatically -> https://github.com/davidjerleke/embla-carousel/pull/423